### DISCUSSION TITLE
Backend Web App "uvicorn" logger not working

### DISCUSSION THREAD

**Discussion by supplemarty** [microsoft/PubSec-Info-Assistant#818] [Morning (EST) Office Hours (Q&A)]
The backend (and enrichment) web apps use the "uvicorn" logger as seen here:
`log = logging.getLogger("uvicorn")
log.setLevel('DEBUG')
log.propagate = True`

When I use this log variable in the backend code for debug tracing like:
`log.debug("HELLO WORLD")`
those log entries do not show up in the log stream NOR application insights but in the enrichment app this same pattern seems to be working fine.

I setup my own logger using the StreamHandler, and it works as I would expect. I cannot see where uvicorn is getting its logging dict configuration in the azure web app configuration. Thoughts?

**Comment by dayland** [comment]
We have not observed this, but have also not explicitly tested it either. If you would like additional assistance on this, please file an Issue and it will get triaged along with other Issues.
